payment-manager/src/main/java/com/qpay/paymentmanager/config/WebClientConfig.java

```diff
@@ -0,0 +1,25 @@
+package com.qpay.paymentmanager.config;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.reactive.function.client.WebClient;
+
+@Configuration
+public class WebClientConfig {
```

Comment:
again think globally. Right now it's only for your target client and it is not extensible for others

Reply:
same as above question